### PR TITLE
[DC-3336] Update vocabulary_upload_qc.py to account for new custom co…

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py [hunk 1]
```diff
@@ -15,27 +15,29 @@
...
 # # + Import packages
 from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
+
 import pandas as pd
 
+# get current custom concepts
+custom_concepts_csv_directory = ".\..\\..\\resource_files\\aou_vocab\\CONCEPT.csv"
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/vocabulary_upload_qc.py:29]: I would suggest retrieving this resources.AOU_VOCAB_PATH

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py [hunk 2]
```diff
@@ -122,16 +124,23 @@
 ORDER BY concept_id
 ''')
 query = tpl.render(vocabulary_dataset_old=vocabulary_dataset_old,
-                   vocabulary_dataset_new=vocabulary_dataset_new,
-                  custom_concepts=custom_concepts)
+                   vocabulary_dataset_new=vocabulary_dataset_new)
 df = execute(client, query, max_rows=True)
 
+missing_items = [item for item in custom_concept_ids if item not in df['concept_id'].values]
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/vocabulary_upload_qc.py:130]: This makes sense, thanks!

**Review by nishanthpp93 — COMMENTED** [review]
Looks good overall! Added some minor suggestions

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py (hunk 1 above)
Comment (brendagutman) [reply data_steward/analytics/cdr_ops/vocabulary_upload_qc.py:29]: !! Thanks for pointing that out!

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**